src/resolve/resolved-dnstls-openssl.c

```diff
@@ -367,6 +367,14 @@ ssize_t dnstls_stream_read(DnsStream *stream, void *buf, size_t count) {
         return ss;
 }
 
+int dnstls_stream_has_buffered_data(DnsStream *stream) {
+        assert(stream);
+        assert(stream->encrypted);
+        assert(stream->dnstls_data.ssl);
+
+        return SSL_has_pending(stream->dnstls_data.ssl);
```

Comment:
The man page says it returns 0 or 1, but for safety `return SSL_has_pending() > 0;`

Reply:
Done.